examples/upload.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function
 from past.builtins import xrange
```

Comment:
This relies on the `future` module (which is unavailable), we should use the builtin `range` instead.